From d10a0f13a51a466860a9ad8367b7224cfb3302f2 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@kernel.org>
Date: Mon, 29 Jun 2020 11:19:21 -0500
Subject: [PATCH 047/120] arm64: dts: agilex: add status to qspi dts node

commit e634accfc9248e603a2a6f094de6bf5be30ffc92 from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit 390a90089db5664bbde3be8a3975a3b9bc56a909]

Add status = "okay" to QSPI node.

Fixes: c4c8757b2d895 ("arm64: dts: agilex: add QSPI support for Intel
Agilex")
Cc: linux-stable <stable@vger.kernel.org> # >= v5.5
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex_socdk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex_socdk.dts b/arch/arm64/boot/dts/intel/socfpga_agilex_socdk.dts
index ac6e51b403d8..96c50d48289d 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex_socdk.dts
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex_socdk.dts
@@ -106,6 +106,7 @@
 };
 
 &qspi {
+	status = "okay";
 	flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-- 
2.31.1

